.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
This is all that really matters, but it might be nice to provide some quick others that might trip people up later, e.g.:
```
.venv
**/__pycache__
```